docs/simple_demo/simple_operation/aggregationDemo.md

```diff
@@ -0,0 +1,41 @@
+[# Demo for obkv-table-client-rs
+Edited by OBKV developers on July 10, 2023.
+
+## Introduction
+obkv-table-client-rs is Rust Library that can access table data from OceanBase storage layer.
+Now we provide an interface to access data from OceanBase, which we will introduce in this document.
+
+***Notice that we will also provide another interface to access data from OceanBase in the future(Like [Mutation](https://github.com/oceanbase/obkv-table-client-java/tree/master/example/simple-mutation)).***
+
+## demo
+### Aggregate
+Aggregate allows the user to get a range of data for ```max()```、```min()```、```count()```、```sum()```、```avg()```.
+A **Aggregate** could get from **ObTableClient** by calling ```aggregate()``` method, then you could customize your aggregate by calling methods in **ObTableAggregation**.
```

Comment:
We should tell users what is the column name of the result.

Reply:
done